docs/use.md

```diff
@@ -714,6 +714,69 @@ Uses plan: `10866183-a775-49e8-96e3-4e7a901e4a79`
 
 
 
+--------------------------------------------------------------------------------
+
+# ![](https://cloud.google.com/_static/images/cloud/products/logos/svg/stackdriver.svg) Stackdriver Profiler
+
+Continuous CPU and heap profiling to improve performance and reduce costs.
+
+ * [Documentation](https://cloud.google.com/profiler/docs/)
+ * [Support](https://cloud.google.com/support/)
+ * Catalog Metadata ID: `00b9ca4a-7cd6-406a-a5b7-2f43f41ade75`
+ * Tags: gcp, stackdriver, profiler
+
+## Provisioning
+
+**Request Parameters**
+
+_No parameters supported._
+
+
+## Binding
+
+**Request Parameters**
+
+_No parameters supported._
+
+**Response Parameters**
+
+ * `Email` _string_ - Email address of the service account
```

Comment:
Because this document is autogenerated, do you mind if I tackle this in a new issue to go through all the docs and make sure they're consistent for the 4.1 release?

Reply:
SGTM.